app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -72,6 +96,18 @@ def rrd_special_issue_set?
     end
   end
 
+  def diagnostic_codes
+    form_disabilities = form.dig('form526', 'form526', 'disabilities')
+    form_disabilities.map { |disability| disability['diagnosticCode'] }
+  end
+
+  # TODO: extract into constants code
+  def forward_to_mas?
+    RapidReadyForDecision::Constants::MAS_DISABILITIES.any? do |i|
+      diagnostic_codes.include?(i)
+    end && diagnostic_codes.size == 1
```

Comment:
I would call `diagnostic_codes.size == 1` first since it's a faster operation.

This:
```ruby
    RapidReadyForDecision::Constants::MAS_DISABILITIES.any? do |i|
      diagnostic_codes.include?(i)
    end
```
could be shortened and easier to comprehend if you did set-intersection or subtracting arrays to determine if any element in one array is an element in another array. Even better, since you know there's only 1 element in `diagnostic_codes` (after checking for `diagnostic_codes.size == 1`), just call `MAS_DISABILITIES.include?(diagnostic_codes.first)`.

Reply:
Oh right, since it's only relevant if it's single-issue. thanks!